src/pytest_benchmark/stats.py

```diff
@@ -168,6 +169,7 @@ def __init__(self, fixture, iterations, options):
         self.group = fixture.group
         self.param = fixture.param
         self.params = fixture.params
+        self.cprofile_stats = fixture.cprofile_stats
 
         self.iterations = iterations
         self.stats = Stats()
```

Comment:
Sounds good, but we should strip the „Benchmark” prefix, no sense to have it everywhere (obvious the stuff is going to be about benchmarks right?).